code/modules/reagents/newchem/toxins.dm

```diff
@@ -120,21 +142,32 @@
 	metabolization_rate = 1
 
 /datum/reagent/neurotoxin2/on_mob_life(var/mob/living/M as mob)
-	if(current_cycle <= 4)
-		M.reagents.add_reagent("neurotoxin2", 1.0)
-	if(current_cycle >= 5)
-		if(prob(5))
-			M.emote("drool")
-		if(M.getBrainLoss() < 60)
-			M.adjustBrainLoss(1*REM)
-		M.adjustToxLoss(1*REM)
-	if(current_cycle >= 9)
-		M.drowsyness = max(M.drowsyness, 10)
-	if(current_cycle >= 13)
-		M.Paralyse(8)
 	switch(current_cycle)
-		if(5 to 45)
-			M.confused = max(M.confused, 15)
+		if(1 to 4)
+			current_cycle++
```

Comment:
It does, in fact, work; the cycle increases and it does not metabolize--once it hits the 5th cycle, it starts metabolizing again and continues to increase its cycle count via `..()`